api-admin/src/test/java/bio/terra/pearl/api/admin/AuthTestUtils.java

```diff
@@ -22,6 +23,7 @@ public static void assertAllMethodsAnnotated(
             .filter(
                 method ->
                     Modifier.isPublic(method.getModifiers())
+                        && !Modifier.isStatic(method.getModifiers())
```

Comment:
This skips over all of the static methods when running `assertAllMethodsAnnotated` - not sure if this is considered "correct" but one of the ext classes had a static method that was just a simple string modifier. I assumed we would not count static methods since they would never have access to the DB.

Reply:
nice, can you put this description in a comment in the code?

Reply:
I ended up making this `@Public` annotated - it seems like a special case and I think it's probably better to be explicit